vscode/syntaxes/toit.tmLanguage.json

```diff
@@ -2,7 +2,7 @@
   "comment": "Don't edit the JSON file, but edit the YAML file instead. Use npx js-yaml < syntaxes/toit.tmLanguage.yaml > syntaxes/toit.tmLanguage.json Use the scope inspector from the command palette with the Developer: Inspect Editor Tokens and Scopes\n",
```

Comment:
Did you edit the YAML file?

Reply:
...
thanks. Updated the yaml file as well.